_includes/v20.1/app/django-basic-sample/views.py

```diff
@@ -0,0 +1,99 @@
+from django.http import JsonResponse, HttpResponse
+from django.utils.decorators import method_decorator
+from django.views.generic import View
+from django.views.decorators.csrf import csrf_exempt
+from django.db import IntegrityError
+
+import json
+import sys
+import time
+
+from .models import *
+
+class PingView(View):
+    def get(self, request, *args, **kwargs):
+        return HttpResponse("python/django", status=200)
+
+def retry_on_exception(num_retries=3, on_failure=HttpResponse(status=500), delay_=0.5, backoff_=1.5):
```

Comment:
maybe we could include a comment or pydoc to explain this a bit: both why we need it, and also document the function parameters/units.